packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -83,7 +83,8 @@ function calculateImageDimensionsAndAspectRatio(file, options) {
 }
 
 function prepareQueue({ file, args }) {
-  const { pathPrefix, ...options } = args
+  const { pathPrefix, duotone, ...rest } = args
+  const options = Object.assign(rest, duotone)
```

Comment:
is this correct ? It seems to me like it would "flatten" `duotone`:

```
// some example
const args = { foo: `bar`, duotone: { wat: true }, a: 5 }
const { pathPrefix, duotone, ...rest } = args
const TEST = Object.assign(rest, duotone)
// TEST = {foo: 'bar', a: 5, wat: true}
```

Reply:
options is only used for the digest, `args` is used for `options` return to should be fine

Reply:
oh, right - then my suggestions would be to rename `options` to something else just to indicate it's not really "options" anymore as shape is different (just to avoid confusion as I had above if someone read it some time later). Ideally with some comment with context

Reply:
@pieh Done